Sources/TensorFlow/Operators/Math.swift

```diff
@@ -408,6 +408,20 @@ internal func _vjpLog<T: TensorFlowFloatingPoint>(
     return (log(x), { v in v / x })
 }
 
+/// Computes the base-two logarithm of the specified tensor element-wise.
+@inlinable
+@differentiable(where T: TensorFlowFloatingPoint)
+public func log2<T: FloatingPoint>(_ x: Tensor<T>) -> Tensor<T> {
+    log(x) / log(Tensor(2))
+}
+
+/// Computes the base-ten logarithm of the specified tensor element-wise.
+@inlinable
+@differentiable(where T: TensorFlowFloatingPoint)
+public func log10<T: FloatingPoint>(_ x: Tensor<T>) -> Tensor<T> {
+    log(x) / log(Tensor(10))
```

Comment:
```suggestion
    log(x) / log(10)
```

Reply:
Doesn't work, see [this comment](https://github.com/tensorflow/swift-apis/pull/214#discussion_r292749224).